### PR TITLE
feat(quorum): per-slot stall_timeout_ms — stop false-killing slow-bursty models

### PR DESCRIPTION
**Pull request by jobordu**
## Problem

The `<500-byte` "header stall" timer in `call-quorum-slot.cjs` was **hardcoded to 30s**. That's fine for fast CLIs, but **false-kills slow-bursty models** routed through a third-party Anthropic-compatible API — **GLM-5.2`[1m]`** (Z.AI) and **MiniMax-M3**: they emit a short preamble (~200–500 bytes) then pause **>30s** while generating. They're *slow, not hung* — but the dispatcher declared `STALL`, cooled them down, and they became permanently `UNAVAIL` in quorum.

Found while making a freshly-curated quorum fleet solid: 2 of 3 new agents were unusable in `/nf:quorum` purely from this 30s threshold.

## Fix

`stallTimeoutFor(provider)` reads `stall_timeout_ms` from the provider entry:
- positive number → use it
- absent / ≤0 / NaN → 30s default (fast slots unchanged)

```js
const STALL_TIMEOUT_MS = stallTimeoutFor(provider);  // was: hardcoded 30000
```

Combined with `has_file_access:false` on slow slots (so the Round-1 prompt tells them to answer from context instead of doing slow repo exploration — a config change, not in this PR), all three new agents go from `UNAVAIL` to clean unanimous votes, **including under concurrent parallel dispatch**:

```
PARALLEL round — all 3 new slots:  29s
  antigravity-1 (Gemini 3.5):  YES
  claude-z-ai   (GLM-5.2):     YES   ← was STALL
  claude-minimax(MiniMax-M3):  YES   ← was STALL
```

## Tests (red-proven)

`bin/call-quorum-slot-stall-timeout.test.cjs` (added to `test:ci`): the helper honors a positive per-slot override, accepts numeric strings, and falls back to 30s for absent/≤0/NaN. **All 5 cases fail against the pre-fix source** (no `stallTimeoutFor` export). Existing `call-quorum-slot` suites (retry/latency/cooldown) green; lint OK.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

* **Bug Fixes**
  * Improved timeout handling so stall limits now respect configured per-provider values instead of always using the default.
  * Added safer fallback behavior for missing, invalid, or non-positive timeout values.

* **Tests**
  * Added coverage for timeout edge cases and regression scenarios.
  * Updated CI test execution to use the latest test target list.

### CODE REVIEW

File: bin/call-quorum-slot.cjs [hunk 1]
```diff
@@ -451,6 +451,17 @@ function buildSpawnArgs(provider, prompt, allowedToolsFlag) {
...
 // ─── Subprocess dispatch ───────────────────────────────────────────────────────
+// STALL-TIMEOUT-01: the "header-only stall" timer (used while a slot has produced
+// < 500 bytes) defaults to 30s — fine for fast slots, but slow-bursty models routed
+// via a third-party Anthropic-compatible API (GLM-5.2[1m], MiniMax-M3) emit a short
+// preamble then pause well past 30s while generating. They are slow, not hung. A
+// per-slot `stall_timeout_ms` in providers.json raises the threshold so they are not
+// false-killed. Any value ≤ 0 / NaN / absent falls back to the 30s default.
+function stallTimeoutFor(provider) {
+  const v = Number(provider && provider.stall_timeout_ms);
+  return v > 0 ? v : 30000;
```
[the remark this answers is not in the document]
Comment (jobordu) [reply bin/call-quorum-slot.cjs]: Fixed in 824b462d — stallTimeoutFor now clamps to TIMEOUT_MAX (2^31-1) so an oversized stall_timeout_ms can't make setTimeout fire immediately. Test covers 3e9/MAX+1/Infinity.
